tests/test_tf_ngmix.py

```diff
@@ -0,0 +1,54 @@
+# This module tests our implementation against ngmix
+from numpy.testing import assert_allclose
+import autometacal
+import ngmix
+
+
+scale = .2
+stamp_size=51
+Ngals = 100
+
+def test_tf_ngmix():
+  """
+  This test generates a simple galaxy and measure moments with ngmix, vs.
+  tf_ngmix.
+  """
+
+  gals, _ = autometacal.data.galaxies.make_data(Ngals=100, img_noise=0.0005,
+                                                 gal_g1=np.random.uniform(-.1,.1,100),
+                                                 gal_g2=np.random.uniform(-.1,.1,100),
+                                                 scale=scale)
+
+  obs_list=[]
+  for gal in gals:
+    obs=ngmix.Observation(gal.numpy(),
+                          jacobian=ngmix.DiagonalJacobian(row=25, 
+                                                          col=25, 
+                                                          scale=.2))
+    obs_list.append(obs)
+
+  pixelss= tf.stack([autometacal.tf_ngmix.make_pixels(gal.numpy(),pix_weights,jake) for gal in gals],axis=0)
```

Comment:
pix_weights and jake I don't think are defined at that point

Reply:
After I changed to a batched test, I introduced these mistakes. :( Will fix it.